### PR TITLE
Add list of plugins with short description

### PR DESCRIPTION
**Pull request by tmichalak**
Addresses issue #76 by adding a list of all implemented plugins with a short summary of what each plugin does and what commands it provides.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
...
+
+Adds several commands that allow for collecting information about cells, nets, pins and ports in the design.
+
+Following commands are added with the plugin:
+* get_cells
+* get_nets
+* get_pins
+* get_ports
+
+### Fasm plugin <a name="fasm"></a>
```
Comment (mithro) [inline README.md:30]:
```suggestion
### FASM plugin <a name="fasm"></a>
```

File: README.md [hunk 2]
```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
...
+4. [Integrate inverters](#integrate_inv)
+5. [Parameters](#parameters) 
+6. [QL IOBs](#ql_iob)
+7. [SDC](#sdc)
+8. [Selection](#selection)
+9. [XDC](#xdc)
 
+## Summary
+
+### Design introspection plugin <a name="design_introspection"></a>
```
Comment (mithro) [inline README.md:20]: I don't think you need these `<a name="design_introspection">` as the headers should already have that.

File: README.md [hunk 3]
```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
...
+* get_ports
+
+### Fasm plugin <a name="fasm"></a>
+
+Writes out the design's [fasm features](https://symbiflow.readthedocs.io/en/latest/fasm/docs/specification.html) based on the parameter annotations on a design cell.
+
+The plugin adds the following command:
+* write_fasm
+
+### Get count plugin <a name="get_count"></a>
```
Comment (mithro) [inline README.md:37]: I wonder if this plugin should be part of the Design introspection plugin above?

File: README.md [hunk 4]
```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
...
+* getparam
+
+### QuickLogic IOB plugin <a name="ql_iob"></a>
+
+[QL IOB plugin](./ql-iob-plugin/) annotates IO buffer cells with information from IO placement constraints.
+
+The plugin adds the following command:
+* quicklogic_iob
+
+### SDC plugin <a name="sdc"></a>
```
Comment (mithro) [inline README.md:66]: In the future, we should split this plugin into two parts;
 (a) Clock manipulation (creation / propagation / etc).
 (b) SDC file reading / writing.

Can you log a github issue about that?

File: README.md [hunk 5]
```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
...
+* read_sdc
+* write_sdc
+* create_clock
+* get_clocks
+* propagate_clocks
+* set_false_path
+* set_max_delay
+* set_clock_groups
+
+### Selection plugin <a name="selection"></a>
```
Comment (mithro) [inline README.md:80]: I think this might want to be part of the design introspection plugin above too?

**Review by mithro — APPROVED** [review]
Great start!

Let's merge and iterate?

File: README.md (hunk 4 above)
Comment (tmichalak) [reply README.md:66]: We already have an issue: #30

File: README.md (hunk 3 above)
Comment (tmichalak) [reply README.md:37]: Opened #79

File: README.md (hunk 5 above)
Comment (tmichalak) [reply README.md:80]: Opened #79

File: README.md (hunk 1 above)
Comment (tmichalak) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (tmichalak) [reply README.md]: Fixed

**Pull request merged**